src/PublicApiAnalyzers/Core/Analyzers/DeclarePublicApiAnalyzer.Impl.cs

```diff
@@ -83,6 +84,8 @@ internal ApiData(ImmutableArray<ApiLine> apiList, ImmutableArray<RemovedApiLine>
 
         private sealed class Impl
         {
+            private const char obliviousMarker = '~';
```

Comment:
Should be pascal cased

Reply:
Will fix. I was relying on editor to guide me on that, but maybe editorconfig is not configured in this repo...